content/sensu-go/6.8/operations/deploy-sensu/secure-postgres.md

```diff
@@ -0,0 +1,374 @@
+---
+title: "Secure PostgreSQL"
+linkTitle: "Secure PostgreSQL"
+guide_title: "Secure PostgreSQL"
+type: "guide"
+description: "Learn how to secure communication between Sensu and PostgreSQL with certificate authentication."
+weight: 65
+version: "6.8"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.8:
+    parent: deploy-sensu
+---
+
+This guide describes how to secure communication between Sensu and the PostgreSQL event store using certificate authentication.
+When deploying Sensu for use outside of a local development environment, you should secure it using transport layer security (TLS).
+
+To learn how to secure communications between Sensu and its agents, read [Generate certificates for your Sensu installation][3] and [Secure Sensu][1].
+
+{{% notice note %}}
+**NOTE**: This guide describes one option for securing communication between Sensu and PostgreSQL and is intended as a starting point.
+Your organization's needs may require a different approach.
+{{% /notice  %}}
+
+## Prerequisites
+
+To use this guide, you must have:
+
+- A running Sensu deployment.
+- A running PostgreSQL instance.
+
+## Install cfssl
+
+The [CloudFlare cfssl][2] toolkit is released as a collection of command-line tools.
+
+If you followed [Generate certificates for your Sensu installation][3], you already downloaded and installed the Cloudflare cfssl toolkit.
+If not, run the following commands:
+
+{{< language-toggle >}}
+
+{{< code "RHEL/CentOS" >}}
+sudo curl -s -L -o /bin/cfssl https://github.com/cloudflare/cfssl/releases/download/v1.6.2/cfssl-bundle_1.6.2_linux_amd64
+
+sudo curl -s -L -o /bin/cfssl-certinfo https://github.com/cloudflare/cfssl/releases/download/v1.6.2/cfssl-certinfo_1.6.2_linux_amd64
+
+sudo chmod +x /bin/cfssl*
+{{< /code >}}
+
+{{< code "Ubuntu/Debian" >}}
+
+# Update apt repos
+sudo apt-get update
+
+# Install cfssl
+sudo apt-get install golang-cfssl
+
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+To verify that cfssl is installed, run:
+
+{{< code shell >}}
+cfssl version
+{{< /code >}}
+
+## Create a Certificate Authority (CA)
+
+Follow these steps to create a CA with cfssl and cfssljson:
+
+1. Create `/etc/sensu/tls` (which does not exist by default):
+{{< code shell >}}
+mkdir -p /etc/sensu/tls
+{{< /code >}}
+
+2. Navigate to the new /etc/sensu/tls directory:
+{{< code shell >}}
+cd /etc/sensu/tls
+{{< /code >}}
+
+3. Create the CA:
+{{< code shell >}}
+echo '{"CN":"Sensu Test CA","key":{"algo":"rsa","size":2048}}' | cfssl gencert -initca - | cfssljson -bare ca -
```

Comment:
Should there be a step in the previous section to install `cfssljson`? I get this response:

```
-bash: cfssljson: command not found
```